.changeset/healthy-mayflies-drop.md

```diff
@@ -0,0 +1,6 @@
+---
+'nuxt-2-composition-api': minor
+'@nacelle/vue': minor
+---
+
+Replaces `@vue/composition-api` with Vue 2.7
```

Comment:
Might be worth adding some details here about compatibility with `@nuxtjs/composition-api` versions. For example, that this version breaks compatibility with `@nuxtjs/composition-api` versions `0.32.0` and below, and that `@nacelle/vue` is now only expected to be compatible with the `0.33.x` versions of `@nuxtjs/composition-api` that are backed by Vue 2.7.

Reply:
Might also be worth noting that `@nacelle/vue` also now requires transpilation config as described in the README.

Reply:
@NWRichmond would you mind giving the updates I made a once over?